data/na/swagger/1-0-5.yaml

```diff
@@ -845,7 +845,7 @@ definitions:
       payment_method:
         type: string
         description: 'The desired method of payment.'
-        enum: ['card', 'token', 'payment_profile', 'cash', 'cheque', 'interac', 'apple_pay', 'android_pay', 'google_pay', '3d_secure_token','processor_token']
+        enum: ['card', 'token', 'payment_profile', 'cash', 'cheque', 'interac', 'apple_pay', 'google_pay', '3d_secure','processor_token']
```

Comment:
any reason 3d_secure_token was changes to 3ds_token ?

Reply:
That seems to have come from another PR that was merged while I was working on this one which caused a conflict. I will change it back and update the PR to resolve the conflict, thanks